### ISSUE TITLE
I can not add the article's cover image there is no image on the journal webpage for articles.

### ISSUE THREAD

**Issue by accakmak** [Vitaliy-1/oldGregg#93]
I entered the back issues in admin panel. Under the issue I chose edit link. After this I chose an article and under article I chose submission link. In article page I entered metada. In submission section I tried to upload a cover image. When I want to save there is an error "Error occurred processing this form. Please enter the references." Bu the references are ok when I look the references link. How can I upload a cover image in an article?

**Comment by accakmak** [comment]
I tried from this menu but i didn't.

My OJS verison is 3.1.1.4 the last.


<img width="960" alt="aaaaa" src="https://user-images.githubusercontent.com/43828509/46922130-3b217200-d00d-11e8-8cbc-f75df95cae6c.png">

I've reviewed the demo site you've submitted, but I can't get a result.

http://ojsdemo.e-medjournal.com/index.php/oldGregg/index

I used this feature in the OJS 3.0 version and worked very well

My journal's web site is tujom.org.

Thanks in advance.
